drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -416,6 +416,26 @@ def form_definitions(filters:)
       scope.order(updated_at: :desc)
     end
 
+    field :form_identifier, Types::Forms::FormIdentifier, null: true do
+      argument :identifier, String, required: true
+    end
+    def form_identifier(identifier:)
+      raise 'Access denied' unless current_user.can_configure_data_collection?
+
+      Hmis::Form::Definition.where(identifier: identifier).current_versions.first
```

Comment:
It looks like with the proposed behavior, if a form is marked "retired" and there is no published or draft form to replace it, it completely disappears from the config tool interface. (neither `formIdentifiers` or `formIdentifeir` query will resolve it).

 I don't think that is correct, we still need to be able to view those forms (and create new drafts of them to publish again). I didn't think about that edge case when spec'ing.   Do you agree w that desired behavior, and am I understanding this code correctly?

Reply:
Yeah, I think so. I don't think those should just disappear.